### ISSUE TITLE
CR to CR-C Process

### ISSUE THREAD

**Issue by Soyvolon** [501stLegionA3/FiveOhFirstDataCore#51]
# Feature
A process page for promotion CRs to CR-Cs after BCT

# Details
As there is not BCT pages/processes yet, a simple form for moving a CR up to CR-C is needed for after they graduate BCT.

**Comment by Zulu-87351** [comment]
Promotion to CR-C after graduation of BCT is automatic.

**Comment by Soyvolon** [comment]
Yes, but there still needs to be an internal system for that to happen. A BCT instructor needs to fill out a form so the system knows who to promote.
